db/health_structure.sql

```diff
@@ -7533,6 +7613,11 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20220407204625'),
 ('20220428183105'),
 ('20220428191717'),
-('20220428192510');
+('20220428192510'),
+('20220616173636'),
+('20220616195501'),
+('20220623172328'),
+('20220721163813'),
+('20220721165009');
```

Comment:
These seems like a lot of migrations but the PR only adds 2? Was it just out of date?

Reply:
Ugh -- that's because my dev has the DB changes for the VPR refactor that has been left out to dry...